e3/electrolyt/run.py

```diff
@@ -29,6 +34,97 @@ def run(self):
                 if hasattr(self, method_name):
                     getattr(self, method_name)()
 
+    def do_build(self):
+        """Run the build primitive after setting up the sandbox."""
+        self.data.data.sandbox = self.sandbox
+        anod_driver = AnodDriver(anod_instance=self.data.data, store=self.store)
+        anod_driver.activate()
+        anod_driver.anod_instance.build_space.create(quiet=True)
+        if not getattr(anod_driver.anod_instance, 'build', False):
+            logger.error('primitive build not implemented')
```

Comment:
All error message should include enough info to debug the issue. Here you need to at least include the name of the spec.